dpbench/benchmarks/black_scholes/black_scholes_numba_dpex_k.py

```diff
@@ -5,20 +5,15 @@
 
 from math import erf, exp, log, sqrt
 
-import numba_dpex as nb
+import numba_dpex as nbdx
```

Comment:
Left a comment elsewhere, but my preference is for `import numba_dpex as dpex`. It is much clearer what package we